logging/src/main/java/tech/pegasys/teku/logging/LoggingConfigurator.java

```diff
@@ -77,25 +80,27 @@ public static synchronized void addLoggersProgrammatically(
 
   private static void addLoggers(final AbstractConfiguration configuration) {
 
-    if (DESTINATION == null) {
+    if (isUninitialized() || noProgrammaticLoggingRequired()) {
       return;
     }
 
     StatusLogger.getLogger().info("Configuring logging for destination: {}", DESTINATION);
```

Comment:
It's slightly confusing that this winds up printing:
`2020-03-31 09:32:18,769 main INFO Configuring logging for destination: DEFAULT_BOTH`

Given we bail out entirely when custom logging is set, it would probably be more helpful to show the logging destination as "file and console" or something.  And if I'm being picky showing where the log file is being written would be super helpful.

Reply:
>show the logging destination as "file and console" or something. And if I'm being picky showing where the log file is being written would be super helpful.

Better output is always a good idea, I'll make an update